src/main/java/ui/UiRenderer.java

```diff
@@ -16,12 +16,12 @@ public class UiRenderer {
         "Wednesday", "Thursday", "Friday", "Saturday"};
 
     private static final int SPACE_COUNT = 10;
-    private static final String SINGLE_HORIZONTAL_DIVIDER = "+" + "-".repeat(SPACE_COUNT + 2);
+    private static final String SINGLE_HORIZONTAL_DIVIDER = "+" + "-".repeat(SPACE_COUNT);
```

Comment:
Good use of the repeat function to automatically add spaces and store them within this final variable